ci-operator/step-registry/openstack/conf/addetcdramdisk/openstack-conf-addetcdramdisk-commands.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+IGNITIONVERSION="2.2.0"
```

Comment:
In 4.6 we started using Ignition spec v3.1, and v3 is not backward compatible with v2 (like python 3 and python 2). In your specific case the config is the same for 2.2 and 3.1, but in general it doesn't work. 

As I know, the latest Ignition, that we use in 4.6+, can't read v2 configs at all. I just tried to do a dry run there with 2.2.0 and got this error:
`couldn't parse config: unsupported config version`

Reply:
I just ran a test on 4.6
./openshift-install version 
./openshift-install 4.6.13
using IGNITIONVERSION="2.2.0"

Reply:
Yeah, it looks like you just merge it with the main ignition config. In this case the version of your config is ignored and your template will work.